maintainers/team-list.nix

```diff
@@ -368,6 +368,7 @@ with lib.maintainers;
       nialov
       sikmir
       willcohen
+      autra
```

Comment:
Would you please move yourself to the top of the members list so we keep it sorted alphabetically ? Thanks